app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -10,6 +10,9 @@ def self.remapping(options)
   end
 
   def self.raw_create_network(ext_management_system, options)
+    # TODO: remove this log line once this uses the task queue, as the task queue has its own logging
+    _log.info "Command: ManageIQ::Providers::Openstack::NetworkManager::" \
+              "CloudNetwork.raw_create_network, Args: #{options.inspect}"
```

Comment:
Could you use `"#{self.class.name}##{__method__}"` instead of hard coding the class/method names in all the log lines?  Maybe even define a `log_prefix` method since these all seem to be common